### PR TITLE
Cpu temperature readings for devices with aml_thermal interface

### PR DESCRIPTION
**Pull request by drieschel**
Adds correct cpu temp readings for Amlogic devices which have the aml_thermal (AML_PLATFORM_THERMAL) interface enabled.

### CODE REVIEW

File: packages/mediacenter/kodi/scripts/cputemp [hunk 1]
```diff
@@ -23,6 +23,11 @@
 
 TEMP=0
 
+isAmlThermal="no";
+if [ -d '/sys/bus/platform/devices/aml_thermal' ]; then
+ isAmlThermal="yes";
```
Comment (MilhouseVH) [inline packages/mediacenter/kodi/scripts/cputemp]: Just a not - needs an extra space for the indent

File: packages/mediacenter/kodi/scripts/cputemp [hunk 2]
```diff
@@ -60,12 +75,12 @@ if [ "$1" = "cpu" -o "$TEMP" = "0" ]; then
   if [ "$TEMP" = "0" -a -f /sys/bus/acpi/devices/LNXTHERM\:00/thermal_zone/temp ]; then
     TEMP=`cat /sys/bus/acpi/devices/LNXTHERM\:00/thermal_zone/temp`
   fi
-
+
```
Comment (MilhouseVH) [inline packages/mediacenter/kodi/scripts/cputemp]: Nit - lose the spaces (and again below)

**Comment by MilhouseVH** [comment]
Don't you also want this in master? Any plans to upstream the kodi change?

**Comment by drieschel** [comment]
@MilhouseVH I updated the cputemp script. Of course I can push it to master, if it makes sense. :).

Didn't think about pushing it to kodi. Should I?

**Comment by MilhouseVH** [comment]
Thanks.

As for sending upstream, that's the ideal situation as it means one less patch for us to maintain, and other distributions benefit also. I'm not sure of the Kodi support situation regarding AML but submitting a Kodi PR should get some good feedback, if not acceptance.

**Comment by MilhouseVH** [comment]
A version of this would also be good for master. If/when it's merged upstream in Kodi we can then drop the Kodi-specific patch.

**Comment by MilhouseVH** [comment]
You should really squash the last 3 commits into one, `git rebase -i HEAD~4`, then change `pick` to `squash` for the last 2 commits, then save, comment out/delete the 2 commit messages, save, and force push with `git push -f`.

**Comment by drieschel** [comment]
I exactly did the rebase like you explained but somehting went wrong I think. What do now?

**Comment by MilhouseVH** [comment]
Hmmm sorry, not sure how that's happened - you seem to have pulled in another commit from somewhere.

Anyway... after a team discussion this won't be accepted for LibreELEC-7.0, so the best option would be to close this PR, and resubmit against master (which is Kodi 17) for LE8.

**Pull request closed without merging**